packages/react-native-aztec/android/build.gradle

```diff
@@ -11,7 +11,7 @@ buildscript {
         espressoVersion = '3.0.1'
 
         // libs
-        aztecVersion = 'v1.6.4'
+        aztecVersion = 'v1.8.0'
```

Comment:
The Aztec library has been updated to incorporate https://github.com/wordpress-mobile/AztecEditor-Android/pull/1059.

Reply:
I'll do the writing flow tests in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6196#issuecomment-1733404065 to double check it's working as expected.